### PR TITLE
ISSUE-1289 Mobile APP app grid

### PR DESCRIPTION
**Pull request by vttranlina**
resolve https://github.com/linagora/tmail-backend/issues/1289

### CODE REVIEW

File: tmail-backend/apps/memory/src/main/conf/linagora-ecosystem.properties [hunk 1]
```diff
@@ -4,4 +4,21 @@
...
+#     }
+#   }
+# }
+# Use `_` as a substitute for spaces in keys for better readability.
+# Example: `key_with_space` can be interpreted as `key with space` in code.
+
+# mobileApps.Twake_Chat.logoURL=https://twake-chat.xyz/tild3837-6430-4761-b135-303536323633__twake-chat_1.svg
+# mobileApps.Twake_Chat.appId=twake-chat
+# mobileApps.Twake_Drive.logoURL=https://twake-drive.xyz/tild6334-3137-4635-b331-636632306164__drive_1.svg
+# mobileApps.Twake_Drive.webLink=https://tdrive.linagora.com
```
Comment (chibenwa) [inline tmail-backend/apps/memory/src/main/conf/linagora-ecosystem.properties:24]: Where is the `/doc` .adoc file about this?

**Review by Arsnael — COMMENTED** [review]
Read it, nothing to add

File: docs/modules/ROOT/pages/tmail-backend/configure/index.adoc [hunk 2]
```diff
@@ -311,3 +311,63 @@ Centralize logging third party like FluentBit can tail from logging’s file the
 
 
 Those logs can then dinamically be parsed by the link:https://grafana.com/oss/loki/[Loki] stack.
+
+== Ecosystem Discovery
```
Comment (vttranlina) [inline docs/modules/ROOT/pages/tmail-backend/configure/index.adoc:315]: IMO, this extension is not a `JMAP extensions for TMail` 
then I think put document here is better than `jmap-extensions` package

**Review by chibenwa — APPROVED** [review]

**Comment by chibenwa** [comment]
Please also plan PR to deploy this change:

 -  cusomise this file in HELM chart
 - CNB preprod AGC (Twake sync, web url tdrive)
 - CNB preprod OVH (Twake sync, web url tdrive)
 - CNB prod (Twake sync, web url tdrive)
 - MU
 - TWP staging (Twake chat, Twake drive web url)
 - TWP prod (Twake chat, Twake drive web url)
 - imap.linagora.com (Twake chat, Twake drive web url, Twake sync)

**Review by Arsnael — APPROVED** [review]

File: docs/modules/ROOT/pages/tmail-backend/configure/ecosystem-discovery.adoc [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+```
+
+The JSON object response sample:
+```json
+{
+    "linShareApiUrl": "https://linshare.linagora.com/linshare/webservice",
+    "linToApiUrl": "https://linto.ai/demo",
+    "linToApiKey": "apiKey",
+    "twakeApiUrl": "https://api.twake.app",
+    "mobileApps": {
```
Comment (hoangdat) [inline docs/modules/ROOT/pages/tmail-backend/configure/ecosystem-discovery.adoc:23]: IMO, it should be 
``` suggestion
    "mobileApps": [
```

**Pull request merged**